### PR TITLE
grpc-js-xds: Log loaded bootstrap info in xDS client (1.4.x)

### PR DESCRIPTION
**Pull request by murgatroid99**
We need this to correlate client logs with TD logs. (PR for 1.4.x)

### CODE REVIEW

File: packages/grpc-js-xds/src/xds-client.ts [hunk 1]
```diff
@@ -340,6 +340,7 @@ export class XdsClient {
         if (this.hasShutdown) {
           return;
         }
+        trace('Loaded bootstrap info: ' + JSON.stringify(bootstrapInfo, undefined, 2));
```
Comment (lidizheng) [inline packages/grpc-js-xds/src/xds-client.ts:343]: Is it possible to print the final bootstrap with the build_version, user_agent_name, server_features fields?

Comment (murgatroid99) [reply packages/grpc-js-xds/src/xds-client.ts:343]: In this code that's not really treated as the "final bootstrap", but I can log the final `Node` messages that will be sent in the ADS and LRS requests.

Comment (murgatroid99) [reply packages/grpc-js-xds/src/xds-client.ts:343]: I added logging for those too.

**Review by lidizheng — APPROVED** [review]
Thanks for adding this log.

**Pull request merged**